tracer/src/Datadog.AutoInstrumentation.NativeLoader/CMakeLists.txt

```diff
@@ -177,6 +177,8 @@ add_library("Datadog.AutoInstrumentation.NativeLoader" SHARED
         ${GENERATED_OBJ_FILES}
         )
 
+target_compile_definitions("Datadog.AutoInstrumentation.NativeLoader" PRIVATE "-D_GLIBCXX_USE_CXX11_ABI=0")
```

Comment:
I don't know what this does but I'll take your word for it 😬

Reply:
It actually fix the Tracer hang issue (Omer did this for the tracer, it was not done for the native loader)

Reply:
oh, of course! And because we previously only used the native loader on Windows, and the issue was on CentOS, this didn't impact us 👍